setup-e2e-enviroment.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+docker compose up -d
+npx json-server --port 8080 -w json-server-data/db.json -m json-server-data/file.js  -q &
```

Comment:
It would be cleaner to have json-server containerized also. Do you see any reason to avoid running json-server on some node.js container?

Reply:
Done.